config/taxonomy/module.values.schema.json

```diff
@@ -9,33 +9,27 @@
             "description": "Type of modules supported by the M4D manager.",
             "enum": ["copy", "read", "write"]
         },
-        "transformation_name": {
-            "description": "Transformation that can be performed on data.",
+        "protocol": {
             "type": "string",
-            "enum": ["mask", "redact", "encrypt"]
-        },
-        "data_level_name": {
-            "type":"string",
-            "enum":["UNKNOWN", "DATASET", "COLUMN", "ROW", "CELL"]
+            "description": "The mechanism via which the data will be accessed.",
+            "enum": ["s3", "kafka", "db2-jdbc", "m4d-arrow-flight"]
         },
-        "data_level_code": {
-            "type":"integer",
-            "description":"0=unknown, 1=dataset, 2=column, 3=row, 4=cell",
-            "minimum": 0,
-            "maximum": 4          
+        "action_name_columns": {
+            "type": "string",
+            "description": "Type of actions possible on columns",
+            "enum": ["RemoveColumn", "RedactColumn", "EncryptColumn", "PeriodicBlackout"]
         },
-        "protocol": {
-            "type":"string",
-            "description": "The mechanism via which the data will be accessed.",
-            "enum":["s3", "kafka", "db2-jdbc", "m4d-arrow-flight"]
+        "action_name_dataset": {
```

Comment:
Future: pick a better name than action_name_dataset

Reply:
Changed the name to allowable_action_dataset